flake.nix

```diff
@@ -51,7 +39,7 @@
                 Security
               ]);
 
-              LIBPG_QUERY_PATH = libpg_query13;
+              LIBPG_QUERY_PATH = final.libpg_query;
```

Comment:
just curious, does final mean like get the the default version in nixpkgs?

Reply:
Yes, that's correct! This changes it back to how it was pre #287.

FWIW, I was able to have the Nix build work without specifying `LIBPG_QUERY_PATH`. But Jade had mentioned in https://github.com/tdbgamer/libpg_query-sys/pull/4 that it wasn't building properly on certain platforms that I don't have a good way of testing at the moment.